interfaces/acados_template/acados_template/acados_layout.json

```diff
@@ -569,6 +569,12 @@
         ]
     },
     "solver_options": {
+        "time_steps": [
```

Comment:
"time_steps" could be a bit confusing (one could think of integrator steps or something else), how about using "shooting_nodes" and "delta_shooting_nodes" or something like that?

Reply:
I was not using `shooting_nodes` in the templates, because they have size N+1 -> you cannot do check against layout :sweat_smile: 
Moreover, there is the option to set `shooting_nodes` and it will be translated into time_steps, both in Matlab and Python, see e.g.:
https://github.com/acados/acados/pull/573/files#diff-b03620fce6e237f180ca3a6ba33659bf

I guess "thinking of integrator steps" is actually a good way to think about it, since they are the step sizes for the integrators..
Thus, I am not fully convinced about changing to `delta_shooting_nodes`.

Reply:
I guess `time_steps` is fine then. It's just that I tend to think about "intermediate" integrator steps when I read `time_steps`, but I guess this is anyway a near-best choice.